dask/dataframe/multi.py

```diff
@@ -355,6 +356,19 @@ def _pad(base, fillby):
     return pd.concat(dfs, axis=axis, join=join)
 
 
+
+def concat_unindexed_dataframes(dfs):
+    name = 'concat-' + tokenize(*dfs)
+
+    dsk = {(name, i): (pd.concat, [(df._name, i) for df in dfs], 1)
```

Comment:
it may better to raise exception during execution if any of df has different length.